File: 프로그래머스/lv0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String s=my_string.replaceAll("[^0-9]","");
        
        for(char ch: s.toCharArray()){
            answer+=Character.getNumericValue(ch);
        }
        return answer;
    }
}

File: 프로그래머스/unrated/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.swift
import Foundation

func solution(_ my_string:String) -> Int {
    
    var sum = 0
    
    for i in my_string { // 한자리 자연수로만 구성
        if i.isNumber {
            sum += Int(String(i))!
        }
    }
    return sum
}